spec/type/lambda_spec.rb

```diff
@@ -3,8 +3,10 @@
 
 describe lambda('my-lambda-function-name') do
   it { should exist }
-  its(:function_name) { should eq 'my-lambda-function-name' }
-  its(:runtime) { should eq 'nodejs' }
+  its(:description) { should eq 'My Lambda Function' }
+  its(:runtime) { should eq 'pyhton2.7' }
```

Comment:
And again. If it's intentional, add a comment or use something that is less similar to a real word.